### DISCUSSION TITLE
Support for GKE Autopilot

### DISCUSSION THREAD

**Discussion by tanmoysinha** [deepfence/ThreatMapper#1080] [General]
Getting started with ThreatMapper OSS after meeting one of your founders at RSAC 2023.  Found it pretty useful while performing security posture assessment for both our cloud K8s clusters and lift-and-shift VM appliances. However, we do have a bunch of GKE autopilot clusters, it seems the Deepfence sensor/agent are not supported in autopilot, not sure why the sensor require hostNetwork privilege on the cluster nodes. Is there a plan to support in headless K8s clusters?

**Comment by shyam-dev** [comment]
Hello @tanmoysinha  - Host network access is used to identify the connection information in/out of the pods. That helps us to provide additional information around the vulnerabilities. Yes, both headless & autopilot is being worked on and we should roll out a release shortly. You are, of course, welcome to tweak the helm chart to your needs of the autopilot cluster. We did have some of our OSS users using this on autopilot clusters with some tweaks to the helm charts. Let me see if I can request them to share the information here.
